decidim-collaborative_texts/app/views/decidim/collaborative_texts/documents/_suggestions_box_template.html.erb

```diff
@@ -0,0 +1,20 @@
+<div data-component="accordion" id="collaborative-texts-box-{{ID}}">
+  <button class="button button__sm button__text-secondary mt-2 mb-4 px-2" data-controls="panel-box-{{ID}}" aria-label="<%= t("decidim.collaborative_texts.document.toggle") %>" aria-expanded="false">
+    <span>
+      <%= icon("award-line", class: "icon icon-pending") %>
+      <%= icon("check-line", class: "icon icon-hover") %>
+      <%= icon("check-double-fill", class: "icon icon-applied") %>
+
+      <%= counter = t("decidim.collaborative_texts.document.suggestions_count", count: '<span class="collaborative-texts-suggestions-box-items-count"></span>').html_safe %>
```

Comment:
Appending `_html` to your key, will convert the text as html safe. You should also change the i18n keys. 
```suggestion
      <%= counter = t("decidim.collaborative_texts.document.suggestions_count_html", count: '<span class="collaborative-texts-suggestions-box-items-count"></span>') %>
```

Reply:
same here